src/overlays/actors/ovl_En_Bee/z_en_bee.c

```diff
@@ -118,121 +118,122 @@ void EnBee_Destroy(Actor* thisx, PlayState* play) {
-    sp3C.z += Math_CosS(this->unk_20C) * 30.0f;
+    Math_Vec3f_Copy(&nextPos, &this->targetPos[this->posIndex]);
+    nextPos.x += Math_SinS(this->targetYaw) * 30.0f;
+    nextPos.z += Math_CosS(this->targetYaw) * 30.0f;
 
-    if (!(this->unk_218 & 1)) {
-        this->unk_20C += (s16)((s32)randPlusMinusPoint5Scaled(1000.0f) + 4000);
+    if (this->instanceId % 2 == 0) {
+        this->targetYaw += (s16)((s32)randPlusMinusPoint5Scaled(1000.0f) + 4000);
     } else {
-        this->unk_20C -= (s16)((s32)randPlusMinusPoint5Scaled(1000.0f) + 4000);
+        this->targetYaw -= (s16)((s32)randPlusMinusPoint5Scaled(1000.0f) + 4000);
     }
 
-    this->unk_210 += 1000;
-    this->actor.velocity.y = Math_SinS(this->unk_210);
+    this->flightHoverOffset += 1000;
+    this->actor.velocity.y = Math_SinS(this->flightHoverOffset);
 
-    if (this->unk_20C > 0x10000) {
-        this->unk_20C = 0;
-        this->unk_214++;
-        this->unk_214 &= 1;
+    if (this->targetYaw > 0x10000) {
+        this->targetYaw = 0;
+        this->posIndex++;
+        this->posIndex &= 1;
     }
 
-    Math_SmoothStepToS(&this->actor.world.rot.y, Math_Vec3f_Yaw(&this->actor.world.pos, &sp3C), 1, 0x7D0, 0);
+    Math_SmoothStepToS(&this->actor.world.rot.y, Math_Vec3f_Yaw(&this->actor.world.pos, &nextPos), 1, 0x7D0, 0);
     Math_ApproachF(&this->actor.speedXZ, 3.0f, 0.3f, 1.0f);
 
-    if ((this->unk_204 == 0) && (this->actor.params != 0)) {
-        func_80B5ABC4(this);
+    if ((this->attackDelayTimer == 0) && (this->actor.params != BEE_BEHAVIOR_IDLE)) {
+        EnBee_SetupAttack(this);
     }
 }
 
-void func_80B5ABC4(EnBee* this) {
-    Animation_Change(&this->skelAnime, &object_bee_Anim_00005C, 1.0f, 0.0f,
-                     Animation_GetLastFrame(&object_bee_Anim_00005C), 0, -10.0f);
-    this->unk_208 = 1;
-    this->actionFunc = func_80B5AC3C;
+void EnBee_SetupAttack(EnBee* this) {
+    Animation_Change(&this->skelAnime, &gBeeFlyingAnim, 1.0f, 0.0f, Animation_GetLastFrame(&gBeeFlyingAnim), 0, -10.0f);
+    this->isHostile = true;
+    this->actionFunc = EnBee_Attack;
 }
 
-void func_80B5AC3C(EnBee* this, PlayState* play) {
+void EnBee_Attack(EnBee* this, PlayState* play) {
     Player* player = GET_PLAYER(play);
-    Vec3f sp88;
+    Vec3f nextPos;
     f32 rnd;
-    f32 phi_fs1;
+    f32 yawOffset;
     s32 i;
 
-    Math_Vec3f_Copy(&sp88, &player->actor.world.pos);
-    phi_fs1 = (this->unk_218 * 0x700) + 0x2000;
+    Math_Vec3f_Copy(&nextPos, &player->actor.world.pos);
+    yawOffset = (this->instanceId * 0x700) + 0x2000;
 
     for (i = 0; i < 2; i++) {
         rnd = randPlusMinusPoint5Scaled(20.0f);
-        sp88.x += Math_SinS((this->actor.yawTowardsPlayer + ((f32)this->unk_20C)) + phi_fs1) * (rnd + 30.0f);
-        sp88.y = (Math_SinS(this->unk_210) * 10.0f) + (player->actor.floorHeight + 40.0f);
+        nextPos.x += Math_SinS((this->actor.yawTowardsPlayer + ((f32)this->targetYaw)) + yawOffset) * (rnd + 30.0f);
```

Comment:
```suggestion
        nextPos.x += Math_SinS((f32)this->actor.yawTowardsPlayer + this->targetYaw + yawOffset) * (rnd + 30.0f);
```
Would match what you did below in the `Cos`.